Magento2/Sniffs/Functions/DiscouragedFunctionSniff.php

```diff
@@ -39,11 +39,11 @@ class DiscouragedFunctionSniff extends ForbiddenFunctionsSniff
         '^call_user_func_array$' => null,
         '^chdir$' => null,
         '^chgrp$' => null,
-        '^chmod$' => null,
+        '^chmod$' => '\Magento\Framework\Filesystem\DriverInterface::changePermissions() or \Magento\Framework\Filesystem\DriverInterface::changePermissionsRecursively()',
```

Comment:
Need to fix `Line exceeds maximum limit of 120 characters; contains 171 characters` here.